.github/workflows/ci.yml

```diff
@@ -179,21 +179,67 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - test_name: Relayed flow
-            artifact_name: relayed
+          # Define new integration tests here. These are executed in parallel in our CI.
+          # If you define the `perf_test_artifact_name` key, the throughput test will
+          # be activated and performance results will be uploaded as artifacts.
+          # If you leave `perf_test_artifact_name` blank, the performance test will
+          # be skipped.
+
+          # Basic connectivity tests
+          - test_name: Client can ping a resource after portal restarts (Direct)
+            setup: echo 'Noop'
+            execute: |
+              docker compose exec -it client timeout 60 \
+              sh -c 'until ping -W 1 -c 10 172.20.0.100 &>/dev/null; do true; done'
+
+              # Restart portal
+              docker compose restart api
```

Comment:
Instead of `restart`, we should `stop` and then `ping`. This just tests that we can reconnect but not that we can ping while the portal is offline.

Reply:
I'll test both conditions